styles.css

```diff
@@ -18,3 +18,39 @@ h1 {
   font-size: 37px;
   color: #d00000;
 }
+
+/* here is stuff about the body  */
+
+body {
+  background-color: #c7c8ca;
+}
+
+section {
+  display: flex;
+}
```

Comment:
```suggestion
```
I think you could remove this, since you only have sections, which have the class `quiz-question-block`, or? So the three lines shouldn't have any effect.